src/main/java/frc/robot/commands/intakerelease/ReleaseCMD.java

```diff
@@ -3,13 +3,24 @@
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import frc.robot.commands.arm.ArmDropCMD;
+import frc.robot.commands.arm.ArmHomeCMD;
 import frc.robot.subsystems.arm.Arm;
 import frc.robot.subsystems.intakerelease.IntakeRelease;
 
 public class ReleaseCMD extends SequentialCommandGroup {
   public ReleaseCMD(IntakeRelease intakerelease, Arm arm) {
-    addCommands(
-        new ArmDropCMD(intakerelease::haveCone, intakerelease::wantsCone, arm),
-        Commands.run(intakerelease::release, intakerelease).withTimeout(0.5));
+    if (intakerelease.wantsCone()) {
```

Comment:
This wantsCone() check will run at construction of ReleaseCMD() which happens in RobotContainer's constructor.

I think you can do this with 
```
addCommands(
   new ConditionalCommand(coneCommand, cubeCommand, intakerelease::wantsCone));
```

Reply:
Code looks good now, but I think we should test it out before merging.